open_ce/utils.py

```diff
@@ -52,6 +52,7 @@
 CONTAINER_TOOLS = ["podman", "docker"]
 DEFAULT_CONTAINER_TOOL = next(filter(lambda tool: os.system("which {} &> /dev/null".format(tool))
                                       == 0, CONTAINER_TOOLS), None)
+NUM_THREAD_POOL = 16
```

Comment:
Is this arbitrary?

Reply:
I tested with several different values. Started to hit diminishing returns after this value. However, in a situation where there are thousands of feedstocks a larger value might help on systems that can support it.

Reply:
I'm wondering if for smaller systems (like a laptop) that have less than 16 hw threads, if this will hit a wall sooner.
16 isn't crazy huge so maybe this is reasonable enough for now.

Reply:
We could always make it an argument also.

Reply:
yeah - or have it check an ENV var...

Reply:
Yeah, if down the road it seems like we need to change it based on the system, we can make it customizable.